app/policies/partner_policy.rb

```diff
@@ -20,7 +20,7 @@ def new?
   def update?
     return true if user.root?
```

Comment:
Can replace this whole function with

`user.root? OR user.admin_for_partner?(record.id) OR user.neighbourhood_admin_for_partner?(record.id)`

Reply:
for some reason this breaks without the return true statements so I'm keeping them in. I do also like how clear it makes the hierarchy so no bad thing!